backend/routers/spiritual.py

```
@@ -481,29 +563,66 @@ async def clear_birth_chart_cache(request: Request):
         stats = await birth_chart_cache.get_cache_statistics()
         return {
             "success": True,
-            "cache_statistics": stats
+            "statistics": stats
         }
+    except HTTPException:
+        raise
     except Exception as e:
-        print(f"[BirthChart] Error getting cache statistics: {e}")
+        logger.error(f"Error getting cache statistics: {e}")
         raise HTTPException(status_code=500, detail="Failed to get cache statistics")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)